test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -304,6 +308,14 @@ class CheckYourAnswersControllerSpec extends SpecBase {
           status(result) shouldBe SEE_OTHER
           redirectLocation(result).get shouldBe controllers.routes.IncompleteSessionDataController.onPageLoad().url
         }
+
+        "the user has not completed the late appeal question when required" in new Setup(AuthTestModels.successfulAuthResult, true) {
+          when(mockSessionService.getUserAnswers(any()))
+            .thenReturn(Future.successful(Some(userAnswers(correctUserAnswers))))
+          val result: Future[Result] = Controller.onPageLoad()(fakeRequestForCrimeJourneyNoReasonableExcuse)
```

Comment:
Could the fakeRequestForCrimeJourneyNoReasonableExcuse be made explicit in the test? Like fakeRequestWithNoLateAppealReason?

Reply:
Done ✅